src/tests/multihost/sssd/testlib/common/qe_class.py

```diff
@@ -191,6 +191,18 @@ def dnf_uninstall(self, package):
                                raiseonerr=False)
         return cmd.returncode
 
+    @property    
```

Comment:
Why is this a '@property'?
'property' typically means a setter exist, not the case here.

Reply:
removed.